remo/dashboard/static/dashboard/js/kpi.js

```diff
@@ -0,0 +1,386 @@
+                            .x(function(d) { return x(d.week); })
+                            .y0(height)
+                            .y1(function(d) { return y(d.casual); });
+
+        var area_active = d3.svg.area()
+                            .x(function(d) { return x(d.week); })
+                            .y0(height)
+                            .y1(function(d) { return y(d.active); });
+
+        $('#graph-people').text('');
+        var svg = d3.select('#graph-people').append('svg')
+                    .attr('width', width + margin.left + margin.right)
+                    .attr('height', height + margin.top + margin.bottom)
+                    .append('g')
+                    .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+
+        var tip = d3.tip()
+                    .attr('class', 'd3-tip')
+                    .offset([35, 0])
+                    .html(function(d) {
+                        return '<strong>Core:</strong> <span style="color:red">'
+                                + d.core + '</span><br>' +
+                                '<strong>Active:</strong> <span style="color:red">'
+                                + d.active + '</span><br>' +
+                                '<strong>Casual:</strong> <span style="color:red">'
+                                + d.casual + '</span><br>' +
+                                '<strong>Inactive:</strong> <span style="color:red">'
+                                + d.inactive + '</span><br>' +
+                                '<strong>Total:</strong> <span style="color:red">'
+                                + d.active_total + '</span><br>';
+                    });
+
+        svg.call(tip);
+
+        if ((typeof(key)!='undefined') && (typeof(value)!='undefined')) {
+            var apiurl = '/api/kpi/people/?format=json&' + key + '=' + value;
+        } else {
+            var apiurl = '/api/kpi/people/?format=json';
+        }
+        d3.json(apiurl, function(error, data) {
+            $('#people-total').text(data.total);
+            $('#people-total_quarter').text(data.quarter_total);
+            $('#people-total_week').text(data.week_total);
+            $('#people-percentage_week').text(data.week_growth_percentage);
+            $('#people-percentage_quarter').text(data.quarter_growth_percentage);
+
+            var weeks = data.total_per_week;
+
+            weeks.forEach(function(d) {
+                d.week = +d.week;
+                d.core = +d.core;
+                d.active = +d.active;
+                d.casual = +d.casual;
+                d.inactive = +d.inactive;
+                d.active_total = d.core + d.active + d.casual + d.inactive;
+            });
+
+            var formatPercent = d3.format('%U');
+
+            x.domain([1, d3.max(weeks, function(d) { return d.week; })]);
+            y.domain([0, d3.max(weeks, function(d) { return d.active_total; })]);
+
+            svg.append("path")
```

Comment:
Here we could iterate over a list of touples like ``(<class-name>, <var-name>)``.